db/v-2/tables/lnk-contest-execution.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+  <changeSet id="lnk-contest-execution-1" author="sanyavertolet" context="dev or prod">
```

Comment:
why id has suffix `-1`?

Reply:
It is planned to have several changeSets here, so this is just for naming.
As everywhere else